packages/location-field/src/index.tsx

```diff
@@ -0,0 +1,758 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+// @ts-ignore Not Typescripted Yet
+import coreUtils from "@opentripplanner/core-utils";
+import getGeocoder from "@opentripplanner/geocoder";
+// @ts-ignore Not Typescripted Yet
+import LocationIcon from "@opentripplanner/location-icon";
+import React, { useEffect, useState, useRef } from "react";
+import { Ban } from "@styled-icons/fa-solid/Ban";
+import { Bus } from "@styled-icons/fa-solid/Bus";
+import { ExclamationCircle } from "@styled-icons/fa-solid/ExclamationCircle";
+import { LocationArrow } from "@styled-icons/fa-solid/LocationArrow";
+import { Search } from "@styled-icons/fa-solid/Search";
+import { Times } from "@styled-icons/fa-solid/Times";
+import { throttle } from "throttle-debounce";
+import { useIntl, FormattedMessage } from "react-intl";
+// eslint-disable-next-line prettier/prettier
+import type { Location, LocationFieldProps, ResultType } from "./types";
+
+import {
+  GeocodedOptionIcon,
+  Option,
+  TransitStopOption,
+  UserLocationIcon
+} from "./options";
+import * as S from "./styled";
+import { generateLabel, getCombinedLabel } from "./utils";
+
+// FIXME have a better key generator for options
+let optionKey = 0;
+
+function DefaultLocationIcon({
+  locationType
+}: {
+  locationType: string;
+}): React.ReactElement {
+  return <LocationIcon size={13} type={locationType} />;
+}
+
+const LocationField = ({
+  autoFocus = false,
```

Comment:
Sort props

Reply:
Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc